tests/src/Interop/StructMarshalling/ReversePInvoke/MarshalSeqStruct/SeqStructDelRevPInvokeNative.h

```diff
@@ -392,8 +392,8 @@ void ChangeNumberSequential(NumberSequential* p)
 
 bool IsCorrectNumberSequential(NumberSequential* p)
 {
-    if(p->i32 != INT_MIN || p->ui32 != 0xffffffff || p->s1 != -0x8000 || p->us1 != 0xffff || p->b != 0 || 
-        p->sb != 0x7f ||p->i16 != -0x8000 || p->ui16 != 0xffff || p->i64 != -1234567890 ||
+    if(p->i32 != INT_MIN || p->ui32 != 0xffffffff || p->s1 != -0x8000 || p->us1 != 0xffff || p->b != 0 ||
+        p->sb != 0x7f ||p->i16 != -0x8000 || p->ui16 != 0xffff || p->i64 != (uint64_t)-1234567890 ||
```

Comment:
This doesn't look right, the p->i64 is signed and you are now comparing it to 32 bit unsigned value. It seems this cast should not be needed at all, the `-1234567890` fits into 32 bit signed int and so comparing it with 64 bit signed int should not generate any warnings.

Reply:
> to 32 bit unsigned value

I am comparing it with 64 bit `unsigned long long` value (`uint64_t`).